packer/scylla-monitor-template.json

```diff
@@ -6,6 +6,7 @@
       "ami_name": "{{ user `monitor_image_name` }}",
       "source_ami": "{{user `aws_source_ami`}}",
       "instance_type": "{{user `aws_instance_type`}}",
+      "aws_regions": "us-west-2,eu-west-2,eu-west-1,eu-central-1,eu-north-1",
```

Comment:
@fruch @roydahan is it required on all these regions? can't we use one specific?

Reply:
IIUC should be identical to what QA is using for testing regions

Reply:
We are using multiple regions, some cases are multi region

And we fall back to other regions, or we won't have enough resources, like spots.

Some instance types are available in specific regions.

So yeah, we need them all